static/js/components/ClassificationForm.jsx

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/no-danger */
+
+import React, { useReducer } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import FormControl from "@material-ui/core/FormControl";
+import MenuItem from "@material-ui/core/MenuItem";
+import { makeStyles } from "@material-ui/core/styles";
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import * as Actions from '../ducks/source';
+
+
+function makeMenuItem(taxonomy, index) {
+  const render_string = `${taxonomy.name} (${taxonomy.version})`;
+
+  return (
+    <MenuItem value={index} key={index.toString()}>
+      {render_string}
+    </MenuItem>
+  );
+}
+
+
+const ClassificationForm = ({ obj_id, taxonomyList }) => {
+  const latest_taxonomyList = taxonomyList.filter((t) => t.isLatest);
```

Comment:
Either snake_case or camelCase, but not both (JS standard is camelCase, but we have occasional exceptions)

Reply:
fixed.